test/vm/micro_benchmarks/main.cpp

```diff
@@ -932,4 +937,87 @@ int main(int argc, char **argv)
             return cd;
         })
         .run_throughput_benchmark();
+
+    BenchmarkBuilder(
+        args,
+        {.title = "store forwarding stall, constant input",
+         .num_inputs = 2,
+         .has_output = true,
+         .iteration_count = 100,
+         .baseline_seq = KernelBuilder<traits>{}
+                             .dup2()
+                             .dup2()
+                             .add()
+                             .dup3()
+                             .dup3()
+                             .add()
+                             .dup4()
+                             .dup4()
+                             .add()
+                             .dup5()
+                             .dup5()
+                             .add()
+                             .dup6()
+                             .dup6()
+                             .add()
+                             .dup7()
+                             .dup7()
+                             .add()
+                             .dup8()
+                             .dup8()
+                             .add()
+                             .dup7()
+                             .dup5()
```

Comment:
Is there a particular reason to use `dup5` here instead of say `dup7`? My initial impression is that `dup7` would be more natural, or perhaps even remove this dup instruction entirely.

Reply:
I just tried a couple until I hit two that were sufficiently far back to both be spilled to memory...